roles/mariadb/tasks/main.yml

```diff
@@ -65,8 +71,3 @@
   template:
     src: local-homepage/config.php.j2
     dest: "{{ playbook_dev_path | realpath }}/local-homepage/config/mariadb.config.php"
-
-- name: create my.cnf.d
```

Comment:
Does it also creates the directory by creating the KEEP_DIR file ?

Reply:
I don't know, I made a first task for the dir and a second for the file (looks at the two first tasks).

```yaml
- name: create my.cnf.d
  file:
    path: /usr/local/etc/my.cnf.d
    state: directory

# https://github.com/Homebrew/brew/issues/5783
- name: create file to keep /usr/local/etc/my.cnf.d/
  file:
    path: /usr/local/etc/my.cnf.d/KEEP_DIR
state: touch
```

Reply:
Nop, but this is already done at the beginning of the file.

If I remember correctly, this duplicate was a hack aiming to fix the problem.